src/Castle.Core/Compatibility/IntrospectionExtensions.cs

```diff
@@ -16,7 +16,8 @@
 
 namespace System.Reflection
 {
-	internal static class IntrospectionExtensions
+	// GVDM: Changing protection levels here! Castle.Services.Compatibility?
+	public static class IntrospectionExtensions
```

Comment:
Why does this need to be public now? It actually can't be made public, it'll break all consumers of our library because of namespaces.